src/Adapters/IndexManagerAdapter.php

```diff
@@ -107,6 +107,17 @@ public function dropIfExists(string $indexName): IndexManagerInterface
         return $this;
     }
 
+    public function dropAll(): IndexManagerInterface
+    {
+        $indicies = array_keys($this->indexManager->get('*,-*.*'));
```

Comment:
Btw, why the pattern is `-*.*` and not `-.*`?

Reply:
It can be both, they return the same results for me. 👍